### PR TITLE
feat: integrate apps

### PR DESCRIPTION
**Pull request by dominguesgm**
Integrate app list in profile

### CODE REVIEW

File: src/app-mediator/AppMediator.js [hunk 1]
```diff
@@ -6,6 +6,8 @@ import Authenticate from '../pages/authenticate';
 import Sign from '../pages/sign';
 import LockScreen from '../shared/components/lock-screen';
 
+const dummyUnlockFn = () => {};
```
Comment (satazor) [inline src/app-mediator/AppMediator.js]: This line was removed.

File: src/app-mediator/AppMediator.js [hunk 2]
```diff
@@ -24,7 +26,7 @@ class AppMediator extends Component {
     }
 
     render() {
-        const { locked/* , pristine*/ } = this.state;
+        const { locked, pristine } = this.state;
```
Comment (satazor) [inline src/app-mediator/AppMediator.js]: Revert this.

File: src/app-mediator/AppMediator.js (hunk 1 above)
Comment (dominguesgm) [reply src/app-mediator/AppMediator.js]: My bad, wasn't sure it was needed

File: src/shared/components/wallet-enclave/WalletEnclave.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
+import React, { Component, Fragment } from 'react';
```
Comment (satazor) [inline src/shared/components/wallet-enclave/WalletEnclave.js]: This file is supposed to be in app/

Comment (dominguesgm) [reply src/shared/components/wallet-enclave/WalletEnclave.js]: This was not like that in the master branch, are you sure you want me to move the file to app/?

Comment (satazor) [reply src/shared/components/wallet-enclave/WalletEnclave.js]: On master, it's in the right place.. https://github.com/ipfs-shipyard/nomios-web/tree/master/src/app

**Review by satazor — APPROVED** [review]

File: src/shared/components/wallet-enclave/WalletEnclave.js (hunk 3 above)
Comment (dominguesgm) [reply src/shared/components/wallet-enclave/WalletEnclave.js]: Nevermind, misunderstood what you were saying

**Pull request merged**